pkg/drivers/kic/oci/oci.go

```diff
@@ -51,9 +53,18 @@ func DeleteAllContainersByLabel(ociBin string, label string) []error {
 		return nil
 	}
 	for _, c := range cs {
-		cmd := exec.Command(ociBin, "rm", "-f", "-v", c)
```

Comment:
Unrelated to this PR, but I think this function should be named `DeleteContainersByLabel`. I don't understand what the importance of `All` in the name is.